README.md

```diff
@@ -11,7 +11,7 @@ Much more info available via `npm help` once it's installed.
 
 ## IMPORTANT
 
-**You need node v4 or higher to run this program.**
+**You need node v6 or higher to run this program.**
```

Comment:
The text below references "v0.12", but should probably be changed to "v5"

Reply:
@nfischer Done! Thanks for the heads-up!